### DISCUSSION TITLE
issue #253 label

### DISCUSSION THREAD

**Discussion by DimitrisNtarl** [ioniodi/sitegr#284] [Q&A] [answered]
Παρατήρησα πως μόνο το δικό μου issue #253 το οποίο άνοιξα χθες δεν έχει κανένα label οπότε δεν ξέρω και γω πως να δράσω, μήπως σας διέφυγε κάτι; @epidrome @diogenisAl

**Comment by diogenisAl** [comment, marked as answer]
@p19ntar δεν μας διέφυγε κάτι, απλά κάποια στιγμή μέσα στο 24ωρο πρέπει και να κοιμηθούμε
